server/plugin.go

```diff
@@ -20,18 +20,19 @@ const (
 type Plugin struct {
 	plugin.MattermostPlugin
 
-	// configurationLock synchronizes access to the configuration.
```

Comment:
Why did you remove this?

Reply:
I wanted to group the fields to make it easier to see that they were related. Removing the comments probably wasn't the best idea though, so I'll add them back in

Reply:
You can still group, but keep the comments :wink:.